go/rows.go

```diff
@@ -31,29 +31,28 @@ import (
 
 	"go.uber.org/zap"
 
-	"github.com/aws/aws-sdk-go/service/athena/athenaiface"
-
-	"github.com/aws/aws-sdk-go/aws"
-	"github.com/aws/aws-sdk-go/service/athena"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	v2athena "github.com/aws/aws-sdk-go-v2/service/athena"
+	v2athenatypes "github.com/aws/aws-sdk-go-v2/service/athena/types"
```

Comment:
Same question as below, Is there a reason for prefixing these with `v2`?

Reply:
Fixed!